ansible_ai_connect/ai/api/tests/test_feedback_view.py

```diff
@@ -380,26 +379,36 @@ def test_feedback_explanation(self):
             properties = segment_events[0]["properties"]
             self.assertEqual(properties["action"], "1")
 
-    @skip("Schema2 event is not enabled yet")
     def test_feedback_generation(self):
+        action = {
```

Comment:
I'm struggling to understand why the properties in this structure are duplicated.

If it's important that it's duplicated perhaps add a comment so somebody doesn't think to "clean it up" in the future.

Please explain yourself :laughing:

Reply:
Oops, it's me. Let's fix that.